src/engine/strat_engine/blockdev.rs

```diff
@@ -97,7 +98,23 @@ pub fn find_all() -> EngineResult<BTreeMap<PoolUuid, BTreeMap<DevUuid, BlockDev>
     Ok(pool_map)
 }
 
+/// Zero sectors at the given offset
+pub fn wipe_sectors(path: &PathBuf, offset: &Sectors, sector_count: &Sectors) -> EngineResult<()> {
```

Comment:
The method will be more general if it takes a reference to a Path, rather than a PathBuf.